src/style.css

```diff
@@ -0,0 +1,37 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :first-child {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--2 :nth-last-child(n + 4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--3 :nth-last-child(n + 3) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 :nth-last-child(n + 2) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 :nth-last-child(n + 1) {
+  background-image: url(images/star-active.svg);
+}
```

Comment:
This properties are duplicated, combine them into one rule

Reply:
done